### DISCUSSION TITLE
Welcome to OpenCore-Legacy-Patcher Discussions!

### DISCUSSION THREAD

**Discussion by Sudo-Jason-rasp** [Sudo-Jason-rasp/OpenCore-Legacy-Patcher#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pi-Qui** [comment]
For two days I have been trying to get different versions of MacOS on my 2010 MacPro 5,1 with OCLP 0.4.5

Start-ROM-versie:    144.0.0.0.0

SMC-versie (system):    1.39f11

SMC-versie (processor):    1.39f11

Chipsetmodel:    NVIDIA Quadro K2000 with boot screen

Fabrikant:    NVIDIA (0x10de)

devicecode:    0x0ffe

Revisioncode:    0x00a2

ROM-revisie:    3738

Metal: supported, function set macOS GPUFamily1 v4

Diagnostic info: passed

MacOs Catalina will not install, after boot get a black screen.

MacOS Big Sur get to the point of installing and than a black screen.

Mac OS Monterey gets to 96% installation, reboots 2 times and than black screen.

According to Dortania Github I had to do some spoofing and set SMBios to minimal to make OS believe it is version 7,1 solving display port problem

I've been trying for two days in total 24 times, but it doesn't work for me.

Is there a way to make it possible to install Monterey or should I wait for a new OCLP
